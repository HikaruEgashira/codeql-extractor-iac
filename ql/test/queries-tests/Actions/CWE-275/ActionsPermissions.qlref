security/Actions/CWE-275/ActionsPermissions.ql