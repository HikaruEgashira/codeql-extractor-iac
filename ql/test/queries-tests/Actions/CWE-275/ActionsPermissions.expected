| .github/workflows/jobspermission.yml:19:5:26:29 | runs-on ... -latest | Actions Job or Workflow does not set permissions |
| .github/workflows/nopermissions.yml:7:5:13:29 | runs-on ... -latest | Actions Job or Workflow does not set permissions |
